package.json

```diff
@@ -77,7 +77,7 @@
     "files": [
       {
         "path": "packages/api-explorer/dist/index.js",
-        "maxSize": "1mb"
+        "maxSize": "1.2mb"
```

Comment:
Temporarily bumping this up until I'm able to reduce the bundle size of this new OAS form package.